docs/reference/NuGet-Client-SDK.md

```diff
@@ -9,14 +9,51 @@ ms.topic: conceptual
 
 # NuGet Client SDK
 
-The *NuGet Client SDK* refers to a group of NuGet packages centered around [NuGet.Commands](https://www.nuget.org/packages/NuGet.Commands), [NuGet.Packaging](https://www.nuget.org/packages/NuGet.Packaging), and [NuGet.Protocol](https://www.nuget.org/packages/NuGet.Protocol). These packages replace the earlier [NuGet.Core](https://www.nuget.org/packages/NuGet.Core/) library.
+The *NuGet Client SDK* refers to a group of NuGet packages:
+
+* [`NuGet.Protocol`](https://www.nuget.org/packages/NuGet.Protocol) - Used to interact with HTTP and file-based NuGet feeds
+* [`NuGet.Packaging`](https://www.nuget.org/packages/NuGet.Packaging) - Used to interact with NuGet packages. `NuGet.Protocol` depends on this package
+
+You can find the source code for these packages on the [NuGet/NuGet.Client](https://github.com/NuGet/NuGet.Client) GitHub repository.
```

Comment:
```suggestion
You can find the source code for these packages in the [NuGet/NuGet.Client](https://github.com/NuGet/NuGet.Client) GitHub repository.
```
in reads better I think. 

Can you add back the NuGet.Core line?

Reply:
Talked offline. I removed the NuGet.Core line as I'm afraid people will use the package without reading the disclaimer. We'd rather people use NuGet.Protocol package instead.